src/util/time.cc

```diff
@@ -22,6 +22,10 @@ std::string stringTime() {
 bool parseTime(const std::string& in, struct timespec* ts) {
   const char* buf = in.c_str();
   struct tm tm;
+
+  //initialize tm struct to 0s before using it, otherwise mktime will behave unpredictably - Kons
```

Comment:
Please use the same commenting style as in the rest of the code, with capitalization and a space after `//`.